db/re/skill_db.yml

```diff
@@ -19180,9 +19139,11 @@ Body:
     MaxLevel: 5
     Type: Weapon
     TargetType: Attack
-    Range: 3
+    DamageFlags:
+      Critical: true
+    Range: 7
     Hit: Multi_Hit
-    HitCount: -7
+    HitCount: 1
```

Comment:
Reduces attack motion from 7 actions to 1 action.
It was a client change to a single effect.